tests/reference/ast_new-string2-44323ea.stdout

```diff
@@ -1,10 +1,7 @@
-(Module [(Expr (ConstantStr "\"\\" ())) (Expr (ConstantStr "
-\\" ())) (Expr (ConstantStr "\\" ())) (Expr (ConstantStr "   \\" ())) (Expr (ConstantStr "\\ " ())) (Expr (ConstantStr "\
-" ())) (Expr (ConstantStr "\\|" ())) (Expr (ConstantStr "Text\\" ())) (Expr (ConstantStr "\
-" ())) (Expr (Call (Name x Load) [(ConstantStr "\
-class a:
+(Module [(Expr (ConstantStr ""\" ())) (Expr (ConstantStr "
+\" ())) (Expr (ConstantStr "\" ())) (Expr (ConstantStr "   \" ())) (Expr (ConstantStr "\ " ())) (Expr (ConstantStr "\" ())) (Expr (ConstantStr "\|" ())) (Expr (ConstantStr "Text\\" ())) (Expr (ConstantStr "\" ())) (Expr (Call (Name x Load) [(ConstantStr "\class a:
```

Comment:
Here, `""\"`, `"\"`, ... seems to be wrong.

Can you cross check with Python's output and update this? you can use `python -m ast tests/parser/string2.py` to verify the Python's AST.